client/packages/plugins/README.md

```diff
@@ -28,233 +28,123 @@ When running in development mode, the required plugin files are loaded directly
 In production mode the process differs:
 
 - the server provides an endpoint to fetch the list of available plugins
-- the client app fetches the full list on startup and populates the PluginProvider
-- this fetches only the plugin definitions. When a plugin is rendered for the first time, the component is fetched from the server and cached in the PluginProvider.
-- column definitions do not require additional loading, though if the column is using a custom component for rendering, that may require fetching
-
-### Plugin validation
-
-Plugins are signed by the developer and require validation before they are able to be run in production mode - validation is ignored when running in developer mode; an error is logged but the plugin is loaded regardless of validation failures.
-
-See the readme on [plugin validation](../../../server/service/src/plugin/plugin_validation.md) for details of how to sign plugins and the process of loading and validating plugins.
+- the client app fetches the full list on startup, then individually fetches each plugin bundle
+- Using federation module and webpack low level api the plugin is dynamically added to scope
+- When plugin and it's dependencies are resolved it's added to PluginProvider and it becomes available to be used by the core functionality
 
 ### Plugin definitions
 
-Each plugin is defined in a `plugin.json` file which has the following structure
-
-```json
-{
-  "name": "StockDonor",
-  "version": "1.0.0",
-  "components": [
-    {
-      "type": "StockEditForm",
-      "localModule": "StockDonorEditInput",
-      "module": "StockDonorEdit"
-    }
-  ],
-  "columns": [
-    {
-      "type": "Stock",
-      "module": "StockDonorColumn"
-    }
-  ],
-  "dependencies": {
-    "omSupplyVersion": "8.0.0"
-  }
-}
-```
-
-The following types of plugins are currently supported:
-
-**ColumnPlugin**
-- `Stock`
-
-**ComponentPlugin**
-- `StockEditForm`
-- `Stock`
-- `InboundShipmentAppBar`
-- `Dashboard`
-
-
-**Plugin events**
-- `onSaveStockEditForm`
-- `onChangeStockEditForm`
+Plugin version and plugin code are defined in package.json of the plugin, code should be unique across all plugins, so it's a good idea to have something unique in the code. 
```

Comment:
Per suggestion in other PR, I think we would benefit distinguishing `plugin` and `plugin bundle`.

The submodules that we load into `plugin` folder should always be referred to as `plugin bundles`, they define sets of plugins that are merged by the plugin provider

So this section would be `Plugin Bundle definitions` etc...

Reply:
Or, if `bundle` is already an overloaded term given the use of webpack, and the frontend/backend bundling, i could also go for `plugin set`? But I do think we need a distinction, especially since this is intended as external developer facing!

Reply:
Hmm, i feel like plugin is a combination of functionality change, not a single interface override. I hear you, I think if we are changing in docs we should change in most places, I would like to make that change but not sure how and where (do we rename everything plugins bundle, like the 'plugins' export, should that be pluginBundle ?).
Keen to chat about this